internal/graph/cached_resolver_test.go

```diff
@@ -47,10 +47,10 @@ func TestResolveCheckFromCache(t *testing.T) {
 				TupleKey:             tuple.NewTupleKey("document:abc", "reader", "user:XYZ"),
 			},
 			setInitialResult: func(mock *MockCheckResolver, request *ResolveCheckRequest) {
-				mock.EXPECT().ResolveCheck(ctx, request).Times(1).Return(result, nil)
+				mock.EXPECT().ResolveCheck(gomock.Any(), request).Times(1).Return(result, nil)
```

Comment:
Had a tough time trying to get this pass with the tracer wrapping the context. Know `gomock.Any()` isn't a strong assertion but the context does not play a critical role in these tests anyway.